src/components/support-form/helpers/header-styles.ts

```diff
@@ -0,0 +1,11 @@
+import { createStyles, makeStyles } from '@mui/styles'
+import { lighten, Theme } from '@mui/material'
+
+export const useHeaderStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    subtitleText: {
+      color: lighten(theme.palette.primary.dark, 0.1),
+      width: '100%',
+    },
+  }),
+)
```

Comment:
You can import the theme inside the body of `useHeaderStyles` in order to reduce the props that this hook needs.

Example

```suggestion
import theme from 'common/theme'

export const useHeaderStyles = makeStyles(() =>
  createStyles({
    subtitleText: {
      color: lighten(theme.palette.primary.dark, 0.1),
      width: '100%',
    },
  }),
)
```